components/builder-jobsrv/src/data_store.rs

```diff
@@ -207,6 +207,15 @@ impl DataStore {
         migrator.migrate("jobsrv",
                          r#"CREATE INDEX pending_jobs_index_v1 on jobs(created_at) WHERE job_state = 'Pending'"#)?;
 
+        migrator.migrate("jobsrv", r#"ALTER TABLE jobs ADD COLUMN IF NOT EXISTS log_url TEXT DEFAULT NULL "#)?;
```

Comment:
Might not need to do an alter here since we will be starting with fresh DBs in our environments, and other recent schema changes were done directly in the table.